docs/bdk-cli/interface.md

```diff
@@ -101,75 +94,59 @@ These are the main "functions" of the wallet. Most of them are pretty self expla
 
-The `--proxy` flag can be optionally used to specify a SOCKS5 proxy to use when connecting to the Electrum server. Spawning a local Tor daemon and using it as a proxy will allow you to connect to `.onion` Electrum
-URLs. **Keep in mind that only plaintext server are supported over a proxy.**
+The `--retries` flag can be optionally used to specify the number of retries when connecting to
+the Electrum server via the SOCKS5 proxy.
 
-The `--wallet` flag can be used to select which wallet to use, if you have more than one of them. If you get a `ChecksumMismatch` error when you make some changes to your descriptor, it's because it does not
-match anymore the one you've used to initialize the cache. One solution could be to switch to a new wallet name, or delete the cache directory at `~/.bdk-bitcoin` and start from scratch.
+The `--proxy_auth` flag can be optionally used to set the SOCKS5 proxy authentication credentials.
 
-## Subcommands
+The `--stop_gap` flag can be optionally used to stop searching addresses for transactions after finding an unused gap of this length.
 
-| Command | Description |
-| ------- | ----------- |
-| [broadcast](#broadcast)         | Broadcasts a transaction to the network. Takes either a raw transaction or a PSBT to extract |
-| [bump_fee](#bump-fee)         | Bumps the fees of an RBF transaction |
-| [combine_psbt](#combine-psbt)      | Combines multiple PSBTs into one |
-| [create_tx](#create-tx)         | Creates a new unsigned tranasaction |
-| [extract_psbt](#extract-psbt)      | Extracts a raw transaction from a PSBT |
-| [finalize_psbt](#finalize-psbt)     | Finalizes a psbt |
-| [get_balance](#get-balance)       | Returns the current wallet balance |
-| [get_new_address](#get-new-address)   | Generates a new external address |
-| [list_transactions](#list-transactions)      | Lists all the incoming and outgoing transactions of the wallet |
-| [list_unspent](#list-unspent)      | Lists the available spendable UTXOs |
-| [policies](#policies)          | Returns the available spending policies for the descriptor |
-| [public_descriptor](#public-descriptor) | Returns the public version of the wallet's descriptor(s) |
-| [repl](#repl)              | Opens an interactive shell |
-| [sign](#sign)              | Signs and tries to finalize a PSBT |
-| [sync](#sync)              | Syncs with the chosen Electrum server |
+The `--wallet` flag can be used to select which wallet to use, if you have more than one of them. If you get a `ChecksumMismatch` error when you make some changes to your descriptor, it's because it does not match the one you've used to initialize the cache. One solution could be to switch to a new wallet name, or delete the cache directory at `~/.bdk-bitcoin` and start from scratch.
```

Comment:
It's also good to mention here that if no `--wallet` option is given then a random wallet name is generated based on a hash of the descriptor string.